plugins/aztec/.env.example

```
@@ -0,0 +1,25 @@
+P2P_IP=
+P2P_PORT=40400
+
+#Port to run the Aztec Services on
+PORT=8080
+
+#The private key to be used by the publisher.
+#SEQ_PUBLISHER_PRIVATE_KEY=
+
+#Private key of the validator participating in attestation duties. Fund VALIDATOR_ADDRESS with at least 0.01 sepoliaETH.
+VALIDATOR_PRIVATE_KEYS=
+VALIDATOR_ADDRESS=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VALIDATOR_ADDRESS key should go before the VALIDATOR_PRIVATE_KEYS key

(UnorderedKey)